app/helpers/fuzzy_time.rb

```diff
@@ -22,6 +22,8 @@ def ago(timestamp)
                        "about an hour and a half ago"
                      when less_than(23.5 * hours)
                        "about #{(diff / (1 * hours)).round} hours ago"
+                     when less_than(24 * hours)
+                       "about 23.5 to 24 hours ago"
                      end
```

Comment:
It's a good idea to use `else` as the last option in a case statement, that way you don't need to worry if edge cases like "exactly 24 hours ago" are covered.